thefuck/shells/bash.py

```
@@ -0,0 +1,42 @@
+    def _get_history_file_name(self):
+        return os.environ.get("HISTFILE",
+                              os.path.expanduser('~/.bash_history'))
+
+    def _get_history_line(self, command_script):
+        return u'{}\n'.format(command_script)
+
+    def how_to_configure(self):
+        if os.path.join(os.path.expanduser('~'), '.bashrc'):
+            config = '~/.bashrc'
+        elif os.path.join(os.path.expanduser('~'), '.bash_profile'):
+            config = '~/.bashrc'
```

Ruff (0.12.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)